kvbc/include/categorization/updates.h

```diff
@@ -171,10 +172,12 @@ struct Updates {
     return category_updates_;
   }
 
-  std::optional<SharedUpdatesData>& getSharedUpdates() { return shared_update_; }
+  std::optional<SharedKeyValueUpdatesData>& getSharedUpdates() { return shared_update_; }
```

Comment:
I think we should remove that method to avoid users calling it (now that `KeyValueBlockchain` is a friend).